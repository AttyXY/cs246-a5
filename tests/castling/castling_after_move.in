game human human
move e2 e4
move e7 e5
move e1 e2
move f8 b4
move e2 e1
move g8 f6
move f1 b5
move h7 h6
move g1 f3
move h8 h7
move e1 g1
move d1 e2
move h7 h8
move b1 c3
move e8 g8
quit
quit
