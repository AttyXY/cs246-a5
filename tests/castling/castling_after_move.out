WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  ----P---
3  --------
2  PPPPKPPP
1  RNBQ-BNR
   abcdefgh

8  rnbqk-nr
7  pppp-ppp
6  --------
5  ----p---
4  -b--P---
3  --------
2  PPPPKPPP
1  RNBQ-BNR
   abcdefgh

8  rnbqk-nr
7  pppp-ppp
6  --------
5  ----p---
4  -b--P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqk--r
7  pppp-ppp
6  -----n--
5  ----p---
4  -b--P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqk--r
7  pppp-ppp
6  -----n--
5  -B--p---
4  -b--P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

8  rnbqk--r
7  pppp-pp-
6  -----n-p
5  -B--p---
4  -b--P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

8  rnbqk--r
7  pppp-pp-
6  -----n-p
5  -B--p---
4  -b--P---
3  -----N--
2  PPPP-PPP
1  RNBQK--R
   abcdefgh

8  rnbqk---
7  pppp-ppr
6  -----n-p
5  -B--p---
4  -b--P---
3  -----N--
2  PPPP-PPP
1  RNBQK--R
   abcdefgh

8  rnbqk---
7  pppp-ppr
6  -----n-p
5  -B--p---
4  -b--P---
3  -----N--
2  PPPP-PPP
1  RNBQK--R
   abcdefgh

Invalid move. Input another move.
8  rnbqk---
7  pppp-ppr
6  -----n-p
5  -B--p---
4  -b--P---
3  -----N--
2  PPPPQPPP
1  RNB-K--R
   abcdefgh

8  rnbqk--r
7  pppp-pp-
6  -----n-p
5  -B--p---
4  -b--P---
3  -----N--
2  PPPPQPPP
1  RNB-K--R
   abcdefgh

8  rnbqk--r
7  pppp-pp-
6  -----n-p
5  -B--p---
4  -b--P---
3  --N--N--
2  PPPPQPPP
1  R-B-K--R
   abcdefgh

8  rnbqk--r
7  pppp-pp-
6  -----n-p
5  -B--p---
4  -b--P---
3  --N--N--
2  PPPPQPPP
1  R-B-K--R
   abcdefgh

Invalid move. Input another move.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
